src/content/docs/contributing/modules.md

```diff
@@ -773,10 +773,14 @@ The key words "MUST", "MUST NOT", "SHOULD", etc. are to be interpreted as descri
     }
     ```
 
-10. In some cases, STDOUT and STDERR need to be saved to file, for example for reporting purposes. Use the shell command `tee` to redirect the
-    streams to both file and it's original stream. This allows for the streams to be captured by the job scheduler's stream logging capabilities
-    and print them to screen when Nextflow encounters an error. In particular, when using `process.scratch`, the log files may not be preserved when
-    the job scheduler relinquishes the job allocation.
+10. In some cases, STDOUT and STDERR may need to be saved to file, for example for reporting purposes. Use
+    the shell command `tee` to capture the streams but preserve the streams. This allows for the streams to
+    be captured by the job scheduler's stream logging capabilities and print them to screen when Nextflow
+    encounters an error.
+
+    This also makes certain they are captured by Nextflow whereas things written to files only may be lost when
+    the job scheduler relinquishes the job allocation lost if using `process.scratch` - unless those files are
+    designated outputs from the process.
```

Comment:
```suggestion
    This ensures that they are captured by Nextflow. If information is only written to files, it could potentially be lost when the job scheduler gives up the job allocation. 
    This is particularly true if `process.scratch` is used, unless those files are specifically marked as outputs from the process.
```

Reply:
The outputs are lost even if they're designated outputs and `process.scratch` is used. Nextflow copies the designated output files only when the script successfully completes

Reply:
updated the suggestion accordingely